.github/workflows/docs.yml

```diff
@@ -43,28 +43,37 @@ jobs:
         run: make -C docs
 
       - name: Checkout docs site
+        if: github.repository_owner == 'tskit-dev'
         uses: actions/checkout@v2
         with:
           repository: tskit-dev/msprime-docs
           token: ${{ secrets.ADMINBOT_DOCS_TOKEN }}
           path: msprime-docs
 
+      - name: Check for diff
+        if: github.repository_owner == 'tskit-dev'
+        id: diff
+        #The html contains commit hashes which we don't want to count as a change
+        run: |
+          diff -x .buildinfo -r msprime-docs/main docs/_build/html | grep "^[<>]" | grep -v "<title>\|VERSION:" | grep . && echo "::set-output name=change_detected::true" || echo "NO CHANGES"
```

Comment:
If there are changes, they will be printed by the final grep command. Maybe this is clutters the logs and it should instead be `grep . > /dev/null`?

Reply:
I left this in as it might be useful to see the diff sometimes. Log space isn't a problem.